components/signin-button.tsx

```
@@ -45,29 +46,52 @@ const GoogleIcon = () => {
   }
 
   return (
     <AlertDialog open={isOpen} onOpenChange={setIsOpen}>
       <AlertDialogTrigger asChild>
-        <Button className="px-4 text-sm md:px-6 md:text-base">Sign in</Button>
+        <div className="group relative">
+          <Button
+            size="lg"
+            className="relative cursor-pointer overflow-hidden rounded-xl border-2 border-neutral-600 bg-gradient-to-r from-black via-neutral-400 to-black px-8 py-6 text-lg font-bold text-white transition-all duration-500 ease-out hover:scale-105 active:scale-95"
+          >
+            <div className="absolute inset-0 -translate-x-full bg-gradient-to-r from-transparent via-white/20 to-transparent transition-transform duration-1000 ease-out group-hover:translate-x-full"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)